### PR TITLE
update docs

### PR DESCRIPTION
**Pull request by JerryShea**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -1,22 +1,20 @@
...
 
-"Datagrid" **Chronicle Datagrid (CE)** is a high performance, low latency access gateway, it is used to control manage access to your data and your services across a number of machines. CE can handle millions of events per second with latencies of around one microsecond on a single machine. CE allows you to install and configure your own business rules and can be used in a variety of ways:
+**link:https://chronicle.software/queue/[Chronicle Queue]** and also link:https://chronicle.software/queue-enterprise/[Chronicle Queue Enterprise] - persists every event and message, recording these in a memory mapped file. Messages can be written concurrently in the same machine or across a network which ensures no messages are ever lost. CQ can serialize, write, read and deserialize realistic business messages in around two microseconds. This ensures our clients have a low latency complete picture of their trading strategy.
```
Comment (nicktindall) [inline README.adoc]: > Messages can be written concurrently in the same machine or across a network which ensures no messages are ever lost.

I don't think this sentence is very clear, or it's trying to make too many points?

Comment (nicktindall) [reply README.adoc]: My interpretation of what I think it's trying to say is...

> Queues can be written to and read from concurrently, on the same machine or across the network via replication. (something about... persistent messaging means messages delivery is guaranteed even in the case of endpoint downtime?)

Comment (nicktindall) [reply README.adoc]: perhaps
> Using Chronicle Queue for message passing provides an effectively unlimited buffer between producers and consumers and a complete audit trail of every message sent.

**Review by nicktindall — APPROVED** [review]
Some comments, but looks good

File: README.adoc (hunk 1 above)
Comment (JerryShea) [reply README.adoc]: Yep - agree

**Pull request merged**